css/components/_login.scss

```diff
@@ -0,0 +1,72 @@
+.login {
+  font-family: Arial;
+  height: 100%;
+  width: 100%;
+
+  >.login-box {
```

Comment:
Using so many times the `>` selector is dangerous. You are increasing the specificity every time which make harder later on to override the style of an element. Try to remove it and position every selector at the same level.